examples/with-typescript-graphql/pages/_app.tsx

```diff
@@ -1,5 +1,6 @@
+import React from 'react'
```

Comment:
Next.js automatically adds this import to react files, let's remove it from here and from other files where it was also added.

Reply:
Oh, I didn't know that, I'll remove these lines.